requirements.txt

```
@@ -2,3 +2,4 @@ PlexAPI>=4.2.0
 fuzzywuzzy
 pypiwin32
 numpy
+configargparse
```

you should add `ConfigArgParse` to the requirements file.
```suggestion
ConfigArgParse
```
https://pypi.org/project/ConfigArgParse/ this is the package you are using right?

---

yes, that's the one